src/web/components/StickyBottomBanner/CMP.tsx

```diff
@@ -1,11 +1,30 @@
 import React, { useState, useEffect, Suspense } from 'react';
 import { ConsentManagementPlatform } from '@guardian/consent-management-platform/dist/ConsentManagementPlatform';
 import {
-    shouldShow as shouldShow_,
+    init as initCCPA,
+    shouldShow,
     setErrorHandler,
+    checkWillShowUi as checkWillShowCCPAUi,
 } from '@guardian/consent-management-platform';
+import { ccpaApplies } from '@root/src/web/lib/ccpaApplies';
 
-export const shouldShow = shouldShow_;
+// this is the wrong place to be doing this, but it's temporary
+// until we remove the old react CMP component and go 100% sourcepoint
+export const willShowNewCMP = async () => {
+    const useCCPA = await ccpaApplies();
```

Comment:
`ccpaApplies` gets called 2x here, idk how costly it is, but would it not make sense to make sure that it is only ever called once?

Reply:
it's defined in a file further down this PR – it's memoised so the check is only ever performed once

Reply:
in src/web/lib/ccpaApplies.ts